src/opnsense/mvc/app/models/OPNsense/IDS/IDS.xml

```diff
@@ -113,6 +113,10 @@
                         <pass>Pass</pass>
                     </OptionValues>
                 </action>
+                <bypass type="BooleanField">
+                        <default>0</default>
+                        <Required>N</Required>
```

Comment:
better to make this required and upgrade the model version, migration will make sure it's set to 0 after upgrade.

Reply:
I have changed the version and the requirement. Do I have to change something in a migration script, too? I'm unsure about it.

Reply:
nope, it's as simple as this. defaults are part of the standard migration, so adding a new field with a default will enforce an automatic migration. Going to merge this, thanks for the effort!

Reply:
Thank you too for being so supportive.